internal/job/job.go

```diff
@@ -70,6 +71,9 @@ type Job struct {
 // TotalStatus returns the total status of a job, combining the result of fields Status + Passed.
 func (j Job) TotalStatus() string {
 	if Done(j.Status) {
+		if j.Completed {
+			return StateComplete
+		}
 		if j.Passed {
 			return StatePassed
 		}
```

Comment:
What is the precedence? Can a job be passed but not completed? Perhaps you should put `j.Passed` check first?